### PR TITLE
update components

### LINKED ISSUE #4: After initial load, resizing makes clicking on a box changing colour of an unrelated box
Check the video for more info
https://www.loom.com/share/78a75c47170a465fbe9812c8469e8769

### PR DESCRIPTION
**Pull request by mankinchi**
Fix #4

### CODE REVIEW

File: src/components/ColorBox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+			}
+			if (column > 0) {
+				adjacentBoxIds.push(boxesPerRow * row + (column - 1));
+			}
+			if (column < boxesPerRow - 1) {
+				adjacentBoxIds.push(boxesPerRow * row + (column + 1));
+			}
+
+			const randomId = adjacentBoxIds[Math.round(Math.random() * (adjacentBoxIds.length - 1))];
+			requestBoxIdToChangeColour(randomId);
```
Comment (redxzeta) [inline src/components/ColorBox/index.tsx]: can you add the box to change color the user clicks on too?

**Review by redxzeta — CHANGES_REQUESTED** [review]
Nice changes. Feel like there are still things I need to learn

File: src/components/ColorBox/index.tsx (hunk 1 above)
Comment (mankinchi) [reply src/components/ColorBox/index.tsx]: so you want both the box you are clicking on and another random box to change colour?

Comment (redxzeta) [reply src/components/ColorBox/index.tsx]: yea

Comment (mankinchi) [reply src/components/ColorBox/index.tsx]: ok updated

**Pull request merged**